lib/drivers/bayerContourNext.js

```diff
@@ -34,6 +36,9 @@ module.exports = function (config) {
   var cfg = _.clone(config);
   var hidDevice = config.deviceComms;
   var HID_PACKET_SIZE = 64;
+  // With no date & time settings changes available,
+  // timezone is applied across-the-board
+  cfg.tzoUtil = new TZOUtil(cfg.timezone, new Date().toISOString(), []);
```

Comment:
Ah, this doesn't matter with across-the-board timezone application, but just as a reminder in bootstrap-able devices we should try to provide the most recent datum on device timestamp (if available) as 2nd arg of the TZOUtil constructor, not current datetime from `new Date()`. The edge case being if you upload just after Fall Back, say, but the most recent data is from just prior to Fall Back - we want to start with the pre-Fall Back offset (obtained by querying for the offset given the pre-Fall Back timestamp and the timezone), not the post-Fall Back offset.